apps/weshipit/pages/french-react-native-apps.tsx

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable @next/next/no-img-element */
+import { Card, Hero, LinkButton, Prose } from '@weshipit/ui';
+
+import { Layout } from '../components/layout';
+import client from './api/apollo-client';
+import { gql } from '@apollo/client';
+import { InferGetStaticPropsType } from 'next/types';
+
+type FrenchApp = {
+  fields: {
+    name: string;
+    website_url?: string;
+    ios_url?: string;
+    android_url?: string;
+    category: string;
+    logo: {
+      url: string;
+    };
+  };
+};
+
+export async function getStaticProps() {
+  const { data } = await client.query({
+    query: gql`
+      query getFrenchAppsRecords {
+        getFrenchAppsRecords {
+          fields {
+            name
+            android_url
+            ios_url
+            category
+            website_url
+            logo {
+              url
+            }
+          }
+        }
+      }
+    `,
+  });
+  const records: FrenchApp[] = data.getFrenchAppsRecords;
+
+  return {
+    props: {
+      records,
+    },
+  };
+}
+
+export default function FrenchReactNativePage({
+  records,
+}: InferGetStaticPropsType<typeof getStaticProps>) {
+  console.log('🚀 ~ records:', records);
+  const currentYear = new Date().getFullYear();
+  const heroTitle = `French companies using React Native in ${currentYear}`;
+  return (
+    <Layout
+      withHeader
+      withFooter
+      withContainer
+      seoTitle="React Native Usage Among French Companies"
+      seoDescription="Discover how French companies like BlaBlaCar, Doctolib, Ledger, and Shine are harnessing the power of React Native to create robust mobile applications."
+    >
+      <Hero title={heroTitle}></Hero>
+      <Prose size={'xl'} className="my-8">
+        <p>
+          We are building a list of French iOS and Android apps apps that are
+          using React Native in {currentYear}. If you’re working in a French
+          company that uses React Native, <a href={''}>add your app</a> to the
+          list.
+        </p>
+      </Prose>
+      <ul className="my-8">
+        <div className="grid grid-cols-1 gap-4 md:grid-cols-2 xl:grid-cols-3">
+          {records.map((record) => (
+            <Card key={record.fields.name} size={'md'}>
+              <Prose>
+                <div className="flex gap-4">
+                  <img
```

Comment:
Do you think we can use next image here? 
https://nextjs.org/docs/pages/api-reference/components/image